### DISCUSSION TITLE
[Feature] Automatic Albums by Tag/Location/Timeperiod

### DISCUSSION THREAD

**Discussion by keineantwort** [immich-app/immich#14573] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Usually you have to add photos to an album manually. I'd like to have the ability to create an album by selecting one or more tags and maybe a time period or a zone on a map to create an album.
This is great to share photos with the family if you are on a holiday trip. 
i.e. create an album for timeperiod summer break and gps zone South Korea and tag "share". And I can create another album with the same criteria, but additionally the tag "family". So you can share this album with family and friends at home. Every photo you take will be uploaded automatically. After a nice day, you can just swipe through your photos, tag everything you like to share, and the albums will show all new photos from Korea automatically. No need to add the photos to multiple albums automatically.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1673

**Reply by keineantwort** [reply]
@bo0tzz Thanks for pointing me to the duplicate. I did not see that. Is there a more details Roadmap than https://immich.app/roadmap or https://github.com/orgs/immich-app/projects? I could not find an Issue or Roadmap-Item for me to keep an eye on...

**Reply by bo0tzz** [reply]
Nope, that's all

**Comment by sophie4869** [comment]
Sharing a stopgap for anyone who wants trip albums today while this is pending:

I built a small open-source Python CLI — **[Immich Trip Albummer](https://github.com/sophie4869/immich-trip-albums)** — that does location + time-based trip clustering against the Immich API. It finds photos/videos taken away from a configured home city, clusters them into trips (by time gaps and country changes), and creates one album per trip. It's dry-run by default and idempotent on re-runs (albums are matched by a marker in the album description, so renames and re-runs stay safe).

It's an external API client, not a replacement for native support — but it's been working well on my own library, and might be useful to others here in the meantime (or as prior art for the eventual native feature).
